docs/src/man/examples.md

```diff
@@ -29,7 +29,7 @@ Packages that have tagged versions available in `METADATA.jl`.
 - [EzXML.jl](https://bicycle1885.github.io/EzXML.jl/latest/)
 - [Highlights.jl](https://juliadocs.github.io/Highlights.jl/latest/)
 - [IntervalConstraintProgramming.jl](http://dpsanders.github.io/IntervalConstraintProgramming.jl/latest/)
-- [Luxor.jl](https://cormullion.github.io/Luxor.jl)
+- [Luxor.jl](https://github.com/JuliaGraphics/Luxor.jl/stable/)
```

Comment:
Oh, that's a point. For 'stable', the heading is "tagged versions available in METADATA", so I would expect a link to the documentation for that tagged version, rather than some in-development version? But that's just a guess. I'm sure you're right about the URL, I wouldn't know.